### ISSUE TITLE
Create professor model and basic methods

### ISSUE THREAD

**Issue by elyasha** [CodeAndCoffee55/hollow-website#64]

**Comment by elyasha** [comment]
@wdvictor Could you work in this issue ? (Following our last conversation)

**Comment by wdvictor** [comment]
> @wdvictor Could you work in this issue ? (Following our last conversation)

@elyasha Yes I can, but I need to finish the home Page. Or I can leave home Page and do the model this issue, is your call. 
Remembering that @omateusp  is working with me on the home page, but he is unavailable for a couple of days.

**Comment by wdvictor** [comment]
@elyasha , Do we have any requirements documents for these functions? any data model already modeled? These documents will be a helping hand when coding this issue.

**Comment by elyasha** [comment]
@wdvictor Hollow won't allow teacher to be logged in the platform as admins, but we juste need a basic model to show them in a page ''/teachers'. So the basic informations, a teacher profile and methods for sending emails for teachers, updating teacher about the courses and so on. After our 2nd point with Hollow, I will have more information ;)
